lib/datadog/tracing/contrib/redis/quantize.rb

```diff
@@ -60,7 +61,8 @@ def get_verb(command_args)
           def auth_command?(command_args)
             return false unless command_args.is_a?(Array) && !command_args.empty?
 
-            command_args.first.to_sym == :auth
+            verb = command_args.first.to_s
+            %w[AUTH auth].include?(verb)
```

Comment:
The strings `AUTH` and `auth` are frozen, but the array `%w` is not: a new array is created every time this method is called:
```ruby
# frozen_string_literal: true

def x
  puts(%w[AUTH auth].object_id)
end

x # => 60
x # => 80
```

I recommend moving this array to a constant.

Reply:
If you think it looks better, you can also inline this check, given it's only two items:
```suggestion
            verb == 'AUTH' || verb == 'auth'
```